tests/config/config.py

```diff
@@ -0,0 +1,49 @@
+# pylint: disable=missing-module-docstring
+import unittest
+
+from src.module.httprequest import HttpRequest
+
+
+class TestConfiguration(unittest.TestCase):
+    """
+    Test configuration of HttpRequest Module.
+    """
+
+    def setUp(self):
+        """
+        Test get method without not mandatory authentication.
+        """
+
+    def test_missing_http_in_base_url(self):
+        """
+        Should raise ValueError for invalid base url.
```

Comment:
```suggestion
        """Should raise ValueError for invalid base url.
```

The documentation should be explanation of the use case and not what to expect on execution of the use case.